src/pages/api/demands/index.ts

```diff
@@ -85,15 +85,15 @@ export default async function demands(
 ) {
   try {
     const session = await getSession({ req });
-    const email = session?.user?.email;
-    if (!email) {
-      return res.status(204).json([]);
-    }
-
+    const email = session?.user?.email ?? undefined;
     if (req.method === 'GET') {
       return get(res, email);
```

Comment:
That mean that anyone without account can get our demands...
I'm not okay with this !

Reply:
Everyone can see the demands on the map.

Reply:
they can see the fields we want them to see... they don't see the whole demand